package/network/ipv6/ds-lite/files/dslite.sh

```diff
@@ -76,23 +80,22 @@ proto_dslite_setup() {
 	proto_add_data
 	[ -n "$zone" ] && json_add_string zone "$zone"
 
-	json_add_array firewall
-	  json_add_object ""
-	    json_add_string type nat
-	    json_add_string target ACCEPT
-	  json_close_object
-	json_close_array
+	if [ "$tnl_type" = "ds" ]; then
+        json_add_array firewall
+            json_add_object ""
+                json_add_string type nat
+                json_add_string target ACCEPT
+            json_close_object
+        json_close_array
+    fi
```

Comment:
The indentation of this block looks weird. Could you replace 4 spaces with tabs instead?